### PR TITLE
Simplify the code that creates a tar passed to CopyToContainer

### PR DESCRIPTION
**Pull request by myitcv**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -30,6 +31,11 @@ var ForceTLS bool
 var SecureCookie *securecookie.SecureCookie
 var AdminToken string
 
+// Unsafe enables a number of unsafe features when set. It is principally
+// intended to be used in development. For example, it allows the caller to
+// specify the Docker networks to join.
+var Unsafe bool
```
Comment (marcosnils) [inline config/config.go:37]: This is like the `--privileged` docker feature? :joy:

**Comment by myitcv** [comment]
> @myitcv seems like this PR has the networks thing as well? Is this intended?

This PR is stacked on top of https://github.com/play-with-docker/play-with-docker/pull/411 hence you see both commits in this PR. https://github.com/play-with-docker/play-with-docker/pull/411 will need to be merged first, then you will only see the commit from this PR

File: config/config.go (hunk 1 above)
Comment (myitcv) [reply config/config.go:37]: Something like that, yes!

This particular change is made in https://github.com/play-with-docker/play-with-docker/pull/411

**Comment by marcosnils** [comment]
> This PR is stacked on top of #411 hence you see both commits in this PR. #411 will need to be merged first, then you will only see the commit from this PR

Got it..I forgot about the other commit. We should come up with a way to standardize stacked PR's in GH :D

**Comment by myitcv** [comment]
> We should come up with a way to standardize stacked PR's in GH :D

It's one of my bugbears: https://gist.github.com/myitcv/e5c7053bf89b4ed6c4530c316f1bf5a7

**Comment by marcosnils** [comment]
Regarding acf103e54f3f8b844a4c6bb959b59c00b4af71de .. It makes sense since the goroutine doesn't seem to be necessary as we're reading the content of the file synchronously in memory. One thing that it's important to highlight is that if the file is very big, the API server could crash due to OOM. I think we should eventually add an upload limit in the API to avoid this.

**Comment by myitcv** [comment]
Thanks - I'll wait and merge this when the entire stack is ready.

**Pull request merged**